From 8b1f880b17347f3fc7c979a29ba9555b4f1bbd7f Mon Sep 17 00:00:00 2001
From: Olaf Hering <olaf@aepfle.de>
Date: Thu, 21 Mar 2019 14:19:19 +0100
Subject: [PATCH] [cmake] find_package(PkgConfig) sets required variables

Per cmake 3.4 documentation, FindPkgConfig does set the used variables.
There is no need to set them again.

Remove PKG_CONFIG_EXECUTABLE check
Remove setting of PKG_CONFIG_FOUND

Signed-off-by: Olaf Hering <olaf@aepfle.de>
---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec82e148d14f..e60f2753df61 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -92,12 +92,6 @@ set(INCLUDES ${CMAKE_SOURCE_DIR}
              ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR})
 
 find_package(PkgConfig)
-if(KODI_DEPENDSBUILD)
-  if(PKG_CONFIG_EXECUTABLE)
-    set(PKG_CONFIG_FOUND TRUE)
-  endif()
-endif()
-
 find_package(Threads REQUIRED QUIET)
 list(APPEND DEPLIBS ${CMAKE_THREAD_LIBS_INIT})
 
